packages/webapp/src/App.tsx

```diff
@@ -69,7 +69,7 @@ const App = () => {
             return url.pathname;
         }
 
-        return env === 'dev' ? '/dev/getting-started' : '/prod/integrations';
+        return env === 'dev' && isCloud() ? '/dev/getting-started' : `/${env}/integrations`;
```

Comment:
`dev AND isCloud`. Is that even possible?

Reply:
Dev environment 
![image](https://github.com/NangoHQ/nango/assets/1724137/2ffd9e67-f7e3-432b-9fbc-e3fcbe9625b4)

on our cloud

Reply:
got it.